content/blog/_posts/2021-02-fluvio-cloud.md

```diff
@@ -10,158 +10,140 @@ url: /blog/2021/02/announcing-fluvio-cloud-platform
-the future of real-time applications. If you liked the blog, feel free to
-share it around or to come let us know your thoughts! We have a community
-[Discord chat] if you'd like to pop in. If you want to get started building
-with Fluvio, the easiest way to begin is with a [free Fluvio Cloud account]
-and to follow our [getting started guide]!.
-
-[Discord chat]: https://discordapp.com/invite/bBG2dTz
-[free Fluvio Cloud account]: https://cloud.fluvio.io/signup
-[getting started guide]: https://www.fluvio.io/docs/getting-started/
+One of the cool things about Fluvio is that we leverage [Rust's FFI] allowing
+us to reuse core compenents in our non-rust client libraries. This is how we
+were able to build a [node-js client] without too much work.
+
+There are tons of reasons why Fluvio is awesome but this is a post about using
+Fluvio Cloud. :)
+
+[Rust's FFI]: https://doc.rust-lang.org/nomicon/ffi.html#calling-foreign-functions
+[Fluvio CLI]: /docs/getting-started/
+[node-js client]: https://github.com/infinyon/fluvio-client-node
+
+# Using The Fluvio Cloud Platform
+
+We've written plenty of [Fluvio tutorials](/tutorials) but we'd like to show
+how Fluvio cloud acts the same as if you were using it locally or if you host
+it yourself.
+
+## Setup
+Setting up the cloud is very straight forward as mentioned in the introduction
+but we'll reiterate those steps anyway. Once you've verified your account (link
+        in the email), a fluvio instance will be automatically provisioned for
+you. Then to make use of it, just do
+```bash
+fluvio cloud login
+```
+This will propt you for the credentials you used to sign up in the web
+form. It will then store this as a profile setting. You can see this setting by
+doing:
+```bash
+fluvio profile view
+```
+
+If you've already got a local instance of fluvio running, you can easily switch
+the instance the client is using by doing:
+```bash
+fluvio profile switch cloud
+```
+
+## Producing and Consuming a datastream
+
+On a fresh Fluvio instance, you'll need to create some topics:
+```bash
+fluvio topic create hello-fluvio-cloud
+```
+
+Getting data in and out of the cloud is just as easy as all our other tutorials:
+
+In one terminal do:
+```bash
+fluvio consume hello-fluvio-cloud
+```
+
+and in another terminal do:
+
+```bash
+fluvio produce hello-fluvio-cloud
+```
+
+The produce command listens to `stdin` and sends it to fluvio cloud on the
+specified topic. In our case, the topic is `hello-fluvio-cloud`. Now, type
+`Hello fluvio cloud!` (or whatever you'd like) into the producer terminal.
+
+Your two terminals should now look like:
+```bash
+$ fluvio produce hello-fluvio-cloud
+Hello fluvio cloud!
+Ok!
+```
+
+```bash
+$ fluvio consume hello-fluvio-cloud
+Hello fluvio cloud!
+```
+
+Similar to using the commandline, using the [rust
+client](https://crates.io/crates/fluvio) and [nodejs
+client](https://www.npmjs.com/package/@fluvio/client) requires no addional
```

Comment:
. Fluvio Cloud integrated seamlessly the profile switch informs producers and consumer to write and read from the local or cloud environments. No additional coding required.